src/components/dialogs/create-study-dialog.js

```diff
@@ -204,8 +215,33 @@ export const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
         setIsParamsCaseFileDisplayed((oldValue) => !oldValue);
     };
 
-    const [currentParameters, paramsComponent, resetImportParamsToDefault] =
-        useImportExportParams(formatWithParameters, null, null, 'standard');
+    const defaultValues = useMemo(() => {
+        return extractDefaultMap(formatWithParameters);
+    }, [formatWithParameters]);
+
+    const [currentValues, setCurrentValues] = useState({});
+    const currentParameters = {};
+    const onChange = useCallback((paramName, value, isEdit) => {
+        if (!isEdit) {
+            setCurrentValues((prevCurrentValues) => {
+                return {
+                    ...prevCurrentValues,
+                    ...{ [paramName]: value },
+                };
+            });
+        }
+    }, []);
+    const paramsComponent = (
+        <FlatParameters
+            paramsAsArray={formatWithParameters}
+            initValues={currentValues}
+            onChange={onChange}
+            variant="standard"
+        />
+    );
+    const resetImportParamsToDefault = useCallback(() => {
```

Comment:
You don't need to do this, it's done in FlatParameters component,
instead you should simply reset `setCurrentParameters({})` in `handleCloseDialog`